pom.xml

```diff
@@ -93,7 +93,7 @@
   </ciManagement>
 
   <properties>
-    <checkstyle.version>8.40</checkstyle.version>
+    <checkstyle.version>8.41.1</checkstyle.version>
```

Comment:
this should not be changed. when performing a release, we run mvn release which increments the version number automatically.